install.sh

```diff
@@ -0,0 +1,48 @@
+# Constants
+VERSION="v0.4.2"
+SIMPLEX_CHAT="simplex-chat"
+
+######################
+PLATAFORM=""
```

Comment:
```suggestion
PLATFORM=""
```

a small typo here and in some other places

Reply:
Fixed, thanks!